### ISSUE TITLE
race vs. ethnicity?

### ISSUE THREAD

**Issue by postmodern** [CoralineAda/contributor_covenant#38]
I found it odd how race is listed, but not ethnicity? It's a common mistake in North America to conflate the two.

**Comment by CoralineAda** [comment]
Great suggestion, let me fix that.

**Comment by CoralineAda** [comment]
Fixed, thanks so much!

**Comment by DanScharon** [comment]
sorry for necromancing this issue, but my concerns fit the issue description:

In my eyes the highly contested term 'race' should be completely dropped in favor of ethnicity.
Common 'race' definitions are extremely arbitrary and large parts of the anthropological, biological and sociological scientific communities reject the race concept entirely as it doesn't have any taxonomic value as a term. 
This doesn't mean that racism doesn't exist, quite the opposite. I just don't think racist concepts should be reinforced by accepting and using their terms.

Ethnicity is a question of affiliation and also often choice. 'Race' on the other hand is usually only (arbitrarily) externally defined and imposed. While there also other terms in the list that are not influenced by choice (e.g. age) they are at least scientifically not so highly contested.

**Comment by ecerta** [comment]
I respectfully disagree. When it comes down to it, all of these terms are arbitrary in definition in one way or another - any femme trans man or femme nonbinary person can tell you they've felt gender imposed on them externally based on arbitrary markers of gender. And, there's plenty of fat and not fat people who are at a perfectly healthy body size, but hell if that matters based on other people's arbitrary decisions on what the right kind of body should look like in order to continue going through life un-harassed.

People who've experienced racism usually don't say the solution to ending racism is to drop our idea of race entirely; it's to embrace pluralism and accept that race, arbitrary as it is (and again, so is everything else here), is still a way many people carve up humanity and discriminate accordingly. Personally I say it's almost always best to go with what the people being discriminated against are saying would help them, so to that end, let's keep race in the list.

**Comment by DanScharon** [comment]
thank you for your response, @ecerta .

> When it comes down to it, all of these terms are arbitrary in definition in one way or another 

yes, I agree, they are. Because of this, most of the terms open up the possibility for those affected to take the terms, reclaim and redefine them. With 'race' this seems to be incredibly hard. Its ingrained essentialism based on hereditary biological traits doesn't really leave an opening to define race yourself or even choose several. I don't know of any working practical attempts to do so that couldn't also be reformulated with ethnicity as conceptual term.

> People who've experienced racism usually don't say the solution to ending racism is to drop our idea of race entirely;

That is an assumption that is hard to prove, at least in a global context. 
In post-fascist Germany (my context) for example, the term 'race' applied to humans is completely "burned", especially in a scientific context. Nazi race theory (which was just a logical deduction of 19th century pseudoscientific race theory) identified Jews as a race. Your personal traits were inherited and defined by your "blood".
As someone who experienced antisemitic racism my reaction wasn't to just suggest other race definitions but to reject such pseudoscientific attempts altogether. And I am not alone (although I may be "unusual", at least in a global context). 
To sum it up: I don't know of any case where you couldn't also use ethnicity instead of the inherently racist use of 'race'.

**Comment by ecerta** [comment]
I can see we're coming at very different perspectives based on our geography. Here in the U.S., we actually did have several movements to reclaim race-based hate language and discriminatory divisions. In the black community you can see it historically in the Black is Beautiful movement and the Black Power movement (https://en.wikipedia.org/wiki/Black_is_beautiful and https://en.wikipedia.org/wiki/Black_Power) and in a more modern context, the Black Lives Matter movement (https://en.wikipedia.org/wiki/Black_Lives_Matter). For the black community, race tends to be emphasized over ethnicity because for most of the American black community, their heritage was erased once their ancestors were brought to the U.S. as slaves. However, there are lots of black scholars who can talk about this far more eloquently and intelligently than I ever could, so I'd recommend checking them out if you're interested in further reading.

We also threw out the scientific concept of race in the U.S., but it persisted for so long in the form of Jim Crow laws for the black community, blood quantum laws with the Native American community, and redlining for the black, Latinx, Native American and Asian communities that it's really persisted as an identity marker over the years in a way that wasn't really the case over in Europe. As a result we still have HBCU's, affirmative action and Congressional black/Hispanic/Asian caucuses to represent the interests of racial minority communities. I can see how this might not be applicable for a European contributor covenant, but for an American one it's still relevant, I think.

Also, I apologize - when I talked about what minorities tend to ask for after being the victims of racist abuse, I was thinking back to the many American scholars I've read, but I haven't expanded my reading nearly enough to include non-white people outside North America.
